KsApi/Service+RequestHelpers.swift

```diff
@@ -51,10 +51,12 @@ extension Service {
       let task = URLSession.shared.dataTask(with: request) {  data, response, error in
         if let error = error {
           observer.send(error: .requestError(error, response))
+          print("🔴 [KsApi] Failure - Request error \(error.localizedDescription)")
```

Comment:
Is this missing colon? `🔴 [KsApi] Failure - Request error: \(error.localizedDescription)`